eZ/Publish/Core/Persistence/Legacy/Content/Gateway.php

```diff
@@ -144,12 +144,12 @@
      * Returns an array with the relevant data.
      *
      * @param mixed $contentId
-     * @param mixed $version
+     * @param mixed|null $version Current version on null value.
```

Comment:
Good opportunity to make it more clear what we expect here (I'm fine with changing whole docblock of this method for consistency):
```suggestion
     * @param int|null $version Current version on null value.
```